tensorflow/lite/kernels/range.cc

```diff
@@ -120,6 +218,24 @@ TfLiteStatus EvalImpl(TfLiteContext* context, const TfLiteTensor* start,
       CalculateRange<int64_t>(start, delta, output);
       break;
     }
+    case kTfLiteInt8: {
+      CalculateRangeQuantized<int8_t>(start, delta, output);
+      break;
+    }
```

Comment:
Why is unquantized int8 not supported?

Reply:
Hi @vamsimanchala, we have been told that int8 is always quantized so we have added support for quantized int8.

Reply:
I don't think this is generally true. Its totally possible to have a unquantized TFLite model with INT8 dtypes.

> Hi @vamsimanchala, we have been told that int8 is always quantized so we have added support for quantized int8.

How was this communicated?

Reply:
We were working with @rascani  who mentioned to add only the int8 quantized support not only for this range op but for other op's as well for which we have created PRs.